SnakesAndLadder/src/commands/CreateUserCommand.java

```diff
@@ -0,0 +1,22 @@
+package commands;
+
+import controller.UserController;
+import exceptions.InvalidCommandException;
+
+public class CreateUserCommand implements ICommand {
+    UserController userController = new UserController();
+
+    @Override
+    public boolean matches(String command) {
+        if(command.equals("CREATE_USER")) {
+            return true;
+        }
+        return false;
+    }
+
+    @Override
+    public void execute(String command) {
+        String[] userDetails = command.split(" ");
+        userController.createUser(userDetails[1], userDetails[2]);
```

Comment:
Code can be made more readable by assigning userDetails[1] and userDetails[2] to variables